### PR TITLE
Fix podman build handling of --http-proxy flag

### PR DESCRIPTION
**Pull request by rhatdan**
Also fixed a todo for handling of cgroup manager while I was in there.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1848396

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

File: test/e2e/build_test.go [hunk 1]
```diff
@@ -195,4 +195,21 @@ var _ = Describe("Podman build", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	FIt("podman images --http_proxy flag", func() {
```
Comment (mheon) [inline test/e2e/build_test.go:198]: Shouldn't this be 'Podman build --http-proxy flag'?

**Comment by mheon** [comment]
One nit otherwise LGTM

File: test/e2e/build_test.go (hunk 1 above)
Comment (rhatdan) [reply test/e2e/build_test.go]: Yes, plus the FIt is wrong.
Fixed.

**Pull request merged**